### PR TITLE
Task/ow 28966/IAU integration - independent example

### PR DESCRIPTION
**Pull request by alonshp**
https://github.com/user-attachments/assets/3977ef81-4ab0-423e-8c54-6c26e85ca4c4

### CODE REVIEW

File: OpenWeb-SampleApp/Coordinators/MonetizationScreenCoordinator.swift [hunk 1]
```diff
@@ -0,0 +1,72 @@
+//
+//  MonetizationScreenCoordinator.swift
+//  OpenWeb-SampleApp
+//
+//  Created by Anael on 25/11/2024.
+//
+
+import RxSwift
+
+class MonetizationScreenCoordinator: BaseCoordinator<Void> {
```
Comment (alon-ha) [inline OpenWeb-SampleApp/Coordinators/MonetizationScreenCoordinator.swift]: Let's stick to the naming convention (everything is a screen), so let's rename to `MonetizationCoordinator`

File: OpenWeb-SampleApp/Lifecycle/AppDelegate.swift [hunk 2]
```diff
@@ -32,6 +33,13 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
             .take(1)
             .subscribe()
 
+        let exampleStoreURL = "https://apps.apple.com/us/app/spotim-demo/id1234567"
```
Comment (alon-ha) [inline OpenWeb-SampleApp/Lifecycle/AppDelegate.swift]: Let's move this block of code into a function at `AppCoordinator` file.
Call the function, `initialIAUSetup` from `initialSetup` function.

Comment (AnaelPegOW) [reply OpenWeb-SampleApp/Lifecycle/AppDelegate.swift]: done

File: OpenWeb-SampleApp/Coordinators/MonetizationScreenCoordinator.swift (hunk 1 above)
Comment (AnaelPegOW) [reply OpenWeb-SampleApp/Coordinators/MonetizationScreenCoordinator.swift]: done

**Pull request closed without merging**